.github/label-pr-config.yml

```diff
@@ -35,7 +35,7 @@ subSystemLabels:
   /^src\/node_bob*/: c++, quic, dont-land-on-v14.x, dont-land-on-v12.x
 
   # don't label python files as c++
-  /^src\/.+\.py$/: lib / src, needs-ci
+  /^src\/.+\.py$/: python, needs-ci
```

Comment:
This file pattern doesn't match anything in the current `master` / `v12.x-staging` (I presume the same for the ones in between). On `v10.x-staging` it matches
```
src/noperfctr_macros.py
src/notrace_macros.py
```
but Node.js 10 is End-of-Life.

Reply:
should I just remove this line?

Reply:
I don't mind either way (I guess we could leave it in defensively in case a future PR reintroduces a Python file to `src`).